web/styles/app_components.css

```diff
@@ -626,7 +626,13 @@ input.settings_text_input {
 .unread_mention_info:not(:empty) {
     margin-right: 5px;
     margin-left: 2px;
-    opacity: 0.5;
+    opacity: 0.7;
```

Comment:
Why is the opacity being changed here?

Reply:
@karlstolley, I have not made any changes to the opacity. Before the latest update, which I made to resolve conflicts, the opacity was 0.7, as you can see [here](https://github.com/zulip/zulip/compare/2f3d3a8a81be5825c26401f2949f0c9b836ed32c..4ecee575c5888d68c331ab4d6304deb382862a44). The latest commits updated the opacity to 0.5, which, in my opinion, is too light for the @ icon. Therefore, I kept it at 0.7 instead of 0.5.

You can see the difference below:

At 0.7 opacity

![image](https://github.com/user-attachments/assets/8187707b-7576-483f-8d97-bb00ddd17f0d)


At 0.5 opacity

![image](https://github.com/user-attachments/assets/ad162f3f-8e68-4566-9c7f-9a60f19e3780)